.travis.yml

```diff
@@ -24,60 +15,68 @@ stages:
 
 jobs:
   include:
-    - name: "Linters"
+    - name: 'Linters'
       stage: test
       language: python
-      python: "3.6"
+      python: '3.6'
       install:
         - pip3 install --upgrade pycodestyle
       script:
         - pycodestyle --max-line-length=99 --exclude='./python/lookout/sdk' .
 
-    - name: "Python: example integration tests"
+    - name: 'Python: example integration tests'
       stage: test
       language: python
-      python: "3.6"
+      python: '3.6'
       install:
+        - ./_tools/ci-run-bblfsh.sh
         - pip3 install -e python
       script:
         - ./_tools/install-lookout-latest.sh
-        - (python3 language-analyzer.py |& tee -a ./py-analyzer.log)&
-        - ./lookout-sdk review --log-level=debug --from 0a9d1d159d2b0064c32df8d2287b174a91390b1a --to HEAD "ipv4://localhost:2021"
-      after_failure:
-        - echo "test"
-        - cat ./py-analyzer.log
-        - cat ./lookout-install.log
+        - (python3 -u language-analyzer.py |& tee ./py-analyzer.log)&
+        - sleep 5s
+        - ./lookout-sdk review --log-level=debug --from 0a9d1d159d2b0064c32df8d2287b174a91390b1a --to HEAD "ipv4://localhost:2021" |& tee ./sdk.log
+        - grep -v -i "error" ./py-analyzer.log > /dev/null
+        - grep -v -i "error" ./sdk.log > /dev/null
 
-    - name: "Golang: example integration tests"
+    - name: 'Golang: example integration tests'
       stage: test
       language: go
-      go: "1.10"
+      go: '1.10'
       install:
         - go version
         - go get .
       script:
+        - ./_tools/ci-run-bblfsh.sh
         - ./_tools/install-lookout-latest.sh
-        - (go run language-analyzer.go |& tee -a ./go-analyzer.log)&
-        - sleep 1s
-        - ./lookout-sdk review --log-level=debug --from 0a9d1d159d2b0064c32df8d2287b174a91390b1a --to HEAD "ipv4://localhost:2020"
-      after_failure:
-        - cat ./go-analyzer.log
-        - cat ./lookout-install.log
+        - (go run language-analyzer.go |& tee ./go-analyzer.log)&
+        - sleep 5s
+        - ./lookout-sdk review --log-level=debug --from 0a9d1d159d2b0064c32df8d2287b174a91390b1a --to HEAD "ipv4://localhost:2020" |& tee ./sdk.log
```

Comment:
if I understood it, this is going to run the analyzer between 0a9d1d159d2b0064c32df8d2287b174a91390b1a and `HEAD`, but:
- `HEAD` can point to any uncontrolled place,
- 0a9d1d159d2b0064c32df8d2287b174a91390b1a might not exist
I think we should use two commits from a well known fixture repo.

Reply:
Please explain what you mean, I don't understand your concerns:

- why 0a9d1d159d2b0064c32df8d2287b174a91390b1a does not exist? it does.
- what is the problem with head?

Reply:
if `lookout-sdk` repo history is rewritten, `0a9d1d1` commit could be missing.

Reply:
we shouldn't rewrite the history of this repo :-D
it has releases, tags and so on.